Notebooks/beavis-ci.csh

```diff
@@ -93,26 +93,29 @@ git clone git@github.com:LSSTDESC/DC2_Repo.git
 cd DC2_Repo/Notebooks
 
 if ($html) then
-    echo "Making static HTML pages from the available notebooks:"
+    echo "Making static HTML pages from the available master branch notebooks:"
```

Comment:
Adding some documentation on whether someone needs to run this manually and push it or if it is a GH hook/app somehow would be useful.

I finally had to bite the bullet and change my shell from tcsh to bash and I feel your pain, but we probably should get this converted to bash at some point.  The reason (I was shocked to learn!) is that tcsh isn't even installed by default on most linux distributions.  So, it might not be there for a lot of people.